winui/Scheduler/Themes.md

```diff
@@ -0,0 +1,114 @@
+        <td><b>Description</b></td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerBorderBrush</td>
+        <td>Border color of the SfScheduler control.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerBackground</td>
+        <td>Background color of the SfScheduler control.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerBorderThickness</td>
+        <td>Thickness of the border for the SfScheduler control</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerHeaderControlBackground</td>
+        <td>Background color of the header in SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerHeaderControlForeground</td>
+        <td>Foreground color of the header in SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerAllowedViewTypeSelected</td>
+        <td>Background color of the selected allowed view button.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerAllowedViewTypeSelectedForeground</td>
+        <td>Foreground color of the selected allowed view button.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerHeaderControlBorderBrush</td>
+        <td>Border color of the header in SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerHeaderControlBorderThickness</td>
+        <td>Thickness of the border for the header in SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerAllowedViewNavigationButtonPadding</td>
+        <td>Padding for allowed view buttons in SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionSchedulerHeaderAllowedViewButtonCornerRadius</td>
+        <td>Corner radius for allowed view buttons in the header of SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionViewHeaderControlBackground</td>
+        <td>Background color of the view header of SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionViewHeaderControlBorderBrush</td>
+        <td>Border color of the view header of SfScheduler.</td>
+    </tr>
+    <tr>
+        <td>SyncfusionViewHeaderControlBorderThickness</td>
+        <td>Thickness of the border for the view header of SfScheduler.</td>
+    </tr>    
+</table>
+
+You can find the keys for all themes in this [theme resource file.](https://github.com/syncfusion/winui-controls-theme-resource-files/tree/master/Syncfusion.Scheduler.WinUI)
+
+## Customize themes
+
+You can customize the default appearance of themes by overriding key values. Adjust the values of specific keys as required.
+
+{% tabs %}
+{% highlight MainWindow.xaml %}
+
+ <scheduler:SfScheduler>
+
+     <scheduler:SfScheduler.Resources>
+         
+         <ResourceDictionary>
+             <ResourceDictionary.ThemeDictionaries>                 
+                 <ResourceDictionary x:Key="Dark">
+                      <SolidColorBrush x:Key="SyncfusionMonthCellBackground" Color="Beige" />
+                      <SolidColorBrush x:Key="SyncfusionMonthCellForeground" Color="Purple" />
```

Comment:
Change different color to looks UI better.

Reply:
Changed.